### DISCUSSION TITLE
Allow Webpack plugin to generate classNames from .css.js in node_modules

### DISCUSSION THREAD

**Discussion by loliver** [vanilla-extract-css/vanilla-extract#104] [Ideas]
Hey there,

Is it possible for the Webpack plugin to build classNames from `*.css.js` files in `node_modules`? We compile our components before publishing so I'm getting `class="undefined"` when using components.

Cheers,
Oli

**Comment by loliver** [comment]
Nevermind, too many hours last week spent debugging left my brain mush, just fixed half my issue with https://github.com/seek-oss/vanilla-extract/issues/56#issuecomment-828830878

**Comment by mattcompiles** [comment]
This should be resolved by #123

**Reply by loliver** [reply]
Confirmed it's working now.
